cpp/velox/compute/VeloxBackend.cc

```diff
@@ -192,7 +192,7 @@ facebook::velox::cache::AsyncDataCache* VeloxBackend::getAsyncDataCache() const
 }
 
 // JNI-or-local filesystem, for spilling-to-heap if we have extra JVM heap spaces
-void VeloxBackend::initJolFilesystem(const facebook::velox::Config* conf) {
+void VeloxBackend::initJolFilesystem(const std::shared_ptr<const facebook::velox::Config> conf) {
```

Comment:
Let's use const std::shared_ptr<const facebook::velox::Config>& for this func and all below ones

Reply:
Updated.